rgabeflores/rgabeflores.ino

```diff
@@ -0,0 +1,53 @@
+#include <Servo.h>  // Include the Servo library
+
+Servo servoVarName;  // Declare a variable as a Servo object
+int servoPin = 2;  // Declare and initialize a variable to hold the servo's pin
+
+// The setup() function runs once at the startup
+void setup() {
+  // Attach the pin to the servo variable in the setup
+  servoVarName.attach(servoPin);
+}
+
+// The loop() function runs after the startup and runs indefinitely
+void loop() {
+  // Moves servo from one extreme to another
+  for (int pos = 0; pos < 180; pos++) {
+    // The condition simulates a "sprinkler" motion by running the following
+    //   instructions only when the desired position is divisible by 30
+    //   (i.e pos = 30, pos = 60, pos = 90... etc.)
+    if( (pos % 30) == 0){
```

Comment:
Can you fix the whitespace and remove the extra parentheses at line 19? It should look like this:
```c++
if (pos % 20 == 0) {
  doSomething()
}
```
Whitespace is important because it makes your code easier to read and it helps future developers to read your code.

Always keep order of operations in mind when writing code too. Adding too many parenthesis can make confusing code. If you know which operations in a statement will be evaluated first, then the code is cleaner and there is no need for parenthesis. Only add them when it is necessary, otherwise rely on [operator precedence](http://en.cppreference.com/w/cpp/language/operator_precedence).

Other than those two picky things, it looks great!